hint.css

```diff
@@ -0,0 +1,45 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.question-container {
+  width: 80%;
+  margin: auto;
+}
+
+.hint-container {
+  padding: 20px;
+  background-color: lightgoldenrodyellow;
+  border-radius: 8px;
+  justify-content: flex-start;
+}
+
+h1 {
+  color: olive;
+  text-align: center;
+  font-size: 2em;
+  margin-bottom: 20px;
+}
+
+.question {
+  color: olive;
+  font-size: 25px;
+  margin-bottom: 0;
+}
+
+.explanation-pra {
+  border: 2px solid rgb(108, 108, 108);
+  border-radius: 5px;
+  padding: 10px;
+  margin: 10px;
+  min-height: 50px;
+}
+
+#back-btn{
+  display: flex;
+  margin: 20px auto; 
+}
+#back-btn a {
```

Comment:
Please include the second `#back-btn` in the first one. 
Separate the CSS for `#back-btn` and the `a` tag.